src/blueapi/cli/cli.py

```diff
@@ -87,16 +94,47 @@ def get_devices(obj: dict) -> None:
 @controller.command(name="run")
 @click.argument("name", type=str)
 @click.argument("parameters", type=str, required=False)
+@click.option(
+    "-t",
+    "--timeout",
+    type=float,
+    help="Timeout for the plan in seconds. None hangs forever",
```

Comment:
I think we should word this in a more user friendly way given that hanging forever is a legitimate use case, we cannot predict how long some plans will take.
Maybe this...
```suggestion
    help="If supplied, terminates the client if the plan fails to finish within this time limit",
```